src/core/inlines.js

```diff
@@ -109,3 +124,47 @@ export function run(conf) {
     txt.parentNode.replaceChild(df, txt);
   }
 }
+
+/**
```

Comment:
Maybe this should all live in inline-idl-parser.js ?

Reply:
moving again :slightly_smiling_face: